src/lib/components/SiteFooter/index.jsx

```diff
@@ -341,6 +362,7 @@ SiteFooter.defaultProps = {
   siteName: null,
   siteLogo: null,
   marketingSiteBaseUrl: null,
+  enterpriseMarketingLink: {},
```

Comment:
The default here should be `null`. And then, we might want to wrap the rendering of the "edX for Business" link in a conditional such that it only displays if this prop is given to the component.

Reply:
👍